pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java

```diff
@@ -285,10 +504,15 @@ public FunctionState getFunctionState(final @PathParam("tenant") String tenant,
     })
     @Path("/{tenant}/{namespace}/{functionName}/{instanceId}/restart")
     @Consumes(MediaType.APPLICATION_JSON)
-    public void restartFunction(final @PathParam("tenant") String tenant,
-                                    final @PathParam("namespace") String namespace,
-                                    final @PathParam("functionName") String functionName,
-                                    final @PathParam("instanceId") String instanceId) {
+    public void restartFunction(
+            @ApiParam(value = "The function's tenant")
+            final @PathParam("tenant") String tenant,
+            @ApiParam(value = "The function's namespace")
+            final @PathParam("namespace") String namespace,
+            @ApiParam(value = "The function's name")
+            final @PathParam("functionName") String functionName,
+            @ApiParam(value = "The function instanceId (restart all instances if instance-id is not provided")
```

Comment:
```suggestion
            @ApiParam(value = "The function instanceId (if instance-id is not provided, all instances are restarted.")
```

Reply:
You can adopt the pattern and refine other similar cases.
When you say "restart all instances....", readers would guess "who restarts", "the system, user, or sth else."
If you don't want to specify the subject, you can use passive voice.